wp-content/mu-plugins/https.php

```diff
@@ -2,7 +2,7 @@
 /**
  * Plugin Name: HTTPS
  * Description: Fixes and improves HTTPS support.
- * Version:     1.0.0
+ * Version:     2.0.0
```

Comment:
And I don't think this is a breaking change because it's not used anywhere else except on the defined filters on the bottom